org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/ui/tasks/TaskTagPreferencePageFragment2.xtend

```diff
@@ -0,0 +1,64 @@
+package org.eclipse.xtext.xtext.generator.ui.tasks
+
+import com.google.inject.Inject
+import org.eclipse.xtext.Grammar
+import org.eclipse.xtext.xtext.generator.AbstractXtextGeneratorFragment
+import org.eclipse.xtext.xtext.generator.XtextGeneratorNaming
+import org.eclipse.xtext.xtext.generator.model.GuiceModuleAccess
+
+import static extension org.eclipse.xtext.GrammarUtil.*
+import static extension org.eclipse.xtext.xtext.generator.model.TypeReference.*
+
```

Comment:
The result is identical to TaskTagPreferencePageFragment, so maybe its better to add this header instead
`/**
 * @author Stefan Oehme - Initial contribution and API
 * @since 2.6
 */`